generate-event-config.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+: ${srcdir=$(dirname "$0")}
+
+: ${AWK=awk}
+: ${CP=cp}
+: ${MV=mv}
+: ${SED=sed}
+
+set -eu
+
+private_header="$srcdir/evconfig-private.h.in"
+input_file="$srcdir/config.h"
```

Comment:
`config.h` is not `srcdir`, so it will fail if you try to build from non-root directory